bin/lib/git.py

```diff
@@ -1,14 +1,14 @@
 import subprocess
 
 
-def current_branch():
-    return __call(["git", "branch", "--show-current"])
+def has_gh_cli():
+    try:
+        return __call(["gh", "auth", "status"]) != "You are not logged into any GitHub hosts. Run gh auth login to authenticate."
```

Comment:
`gh` is a new tool to me, personally. It looks great, but it might be worth documenting that it's required to run this script. And maybe even having a prompt to install it if this command fails because it's not found? (For my usage of git I guess I was relying on the assumption that the user wouldn't have this app installed unless they also had git install…)

But also bigger picture, in the long run it would probably better to replace these command line calls with the GitHub API, as mentioned in https://app.asana.com/0/1198965985597278/1198965985597278.